src/main/java/org/fundacionjala/coding/dennis/FizzBuzz.java

```diff
@@ -0,0 +1,53 @@
+package org.fundacionjala.coding.dennis;
+
+/**
+ *
+ */
+public  final class FizzBuzz {
+
+    public static final int THREE = 3;
+    public static final int FIVE = 5;
+    public static final int MCM = THREE * FIVE;
+    public static final int LIMIT = 200;
+    /**
+     *
+     */
+    private FizzBuzz() {
+    }
+
+    /**
+     *
+     * @param limit limite
+     */
+
+    public static void fBuzz(int limit) {
+        StringBuilder lString = new StringBuilder();
+        for (int pt = 1; pt <= limit; pt++) {
+            if (pt % MCM == 0) {
+                lString = new StringBuilder("FizzBuzz");
+            } else if (pt % THREE == 0) {
+                lString = new StringBuilder("Fizz");
+            } else if (pt % FIVE == 0) {
+                lString = new StringBuilder("Buzz");
+            } else {
+                lString = new StringBuilder(Integer.toString(pt));
+            }
+
+
+
+//            lString = (pt % MCM == 0) ? (new StringBuilder("FizzBuzz"))
+//                    : ((pt % THREE == 0) ? (new StringBuilder("Fizz"))
+//                    : ((pt % FIVE == 0) ? (new StringBuilder("Buzz"))
+//                    : (new StringBuilder(Integer.toString(pt)))));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)